thorlcr/msort/tsortm.cpp

```diff
@@ -124,7 +120,7 @@ class CSortNode: public SortSlaveMP
         try {
 
             StringBuffer epstr;
-            ActPrintLog(activity, "Connect to %s:%d",endpoint.getIpText(epstr).str(),(unsigned)mpport);
+            ActPrintLog(activity, thorDetailedLogLevel, "Connect to %s:%d",endpoint.getIpText(epstr).str(),(unsigned)mpport);
```

Comment:
::ActPrintLog() ?

Reply:
only needed if ambiguous with class method, there is none in this context.